### PR TITLE
feat: implement a button to switch between new and old UI

### PR DESCRIPTION
**Pull request by shadowusr**
### What's done:
- Go to new UI button in old UI
- Settings panel with go back to old UI
- The choice of UI mode is remembered

Demo report: https://nda.ya.ru/t/dW1Fx8qe78xojb
Demo GIF: https://nda.ya.ru/t/Il_hVvfw78xoku

### CODE REVIEW

File: lib/static/new-ui/app/App.tsx [hunk 1]
```diff
@@ -23,8 +22,7 @@ export function App(): ReactNode {
             element: <SuitesPage/>,
             children: [<Route key={'suite'} path=':suiteId' element= {<SuitesPage/>} />]
         },
-        {title: 'Visual Checks', url: '/visual-checks', icon: Eye, element: <VisualChecksPage/>},
-        {title: 'Info', url: '/info', icon: CircleInfo, element: <InfoPage/>}
```
Comment (KuznetsovRoman) [inline lib/static/new-ui/app/App.tsx:27]: Why did you remove it?

**Review by KuznetsovRoman — APPROVED** [review]

File: lib/static/new-ui/app/App.tsx (hunk 1 above)
Comment (shadowusr) [reply lib/static/new-ui/app/App.tsx:27]: I plan to implement info as a panel instead of a page. But right now it's empty so there's no point in misleading users, i will add it back when it'll be ready.

File: lib/static/new-ui/components/MainLayout/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import {PanelId} from '@/static/new-ui/components/MainLayout/index';
+
+interface FooterProps {
+    visiblePanel: PanelId | null;
+    onFooterItemClick: (item: GravityMenuItem) => void;
+}
+
+export function Footer(props: FooterProps): ReactNode {
+    const isInitialized = useSelector(getIsInitialized);
+    const [isHintVisible, setIsHintVisible] = useState<undefined | boolean>(undefined);
```
Comment (KuznetsovRoman) [inline lib/static/new-ui/components/MainLayout/Footer.tsx:21]: I dont like explicit undefined here
Lets use "false" or "null" instead of undefined here

File: lib/static/new-ui/components/UiModeHintNotification/index.module.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+    80% {
+        filter: blur(0px);
+        transform: scale(1.02);
+    }
+
+    100% {
+        filter: blur(0px);
+        transform: scale(1);
+        visibility: visible;
```
Comment (KuznetsovRoman) [inline lib/static/new-ui/components/UiModeHintNotification/index.module.css:20]: Seems like "visibility: visible" does nothing here

File: lib/static/new-ui/components/UiModeHintNotification/index.module.css [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    border-radius: 10px;
+    color: rgba(255, 255, 255, .9);
+    fill: rgba(255, 255, 255, .9);
+    width: 700px;
+    display: flex;
+    gap: 8px;
+    box-shadow: 0 0 16px 0 #00000036;
+    align-items: center;
+
+    visibility: hidden;
```
Comment (KuznetsovRoman) [inline lib/static/new-ui/components/UiModeHintNotification/index.module.css:58]: Can we remove "visibility: hidden" from here and from "notification-appear" animation?
Feels weird, when "container" class is hidden by default (even when if it is notification hint container)

**Review by KuznetsovRoman — APPROVED** [review]

File: lib/static/new-ui/components/UiModeHintNotification/index.module.css (hunk 3 above)
Comment (shadowusr) [reply lib/static/new-ui/components/UiModeHintNotification/index.module.css:20]: No, it's crucial here. When using animation-fill-mode forwards, setting visibility at the final stage forces element to stay visible after animation end. If we remove this line, notification will abruptly disappear as soon as animation ends.

File: lib/static/new-ui/components/UiModeHintNotification/index.module.css (hunk 4 above)
Comment (shadowusr) [reply lib/static/new-ui/components/UiModeHintNotification/index.module.css:58]: We can't, discussed this in chat
